python_modules/libraries/dagster-dbt/dagster_dbt_tests/cli/test_resources_v2.py

```diff
@@ -25,6 +26,58 @@ def test_dbt_cli(global_config: List[str], command: str) -> None:
     assert dbt_cli_task.process.returncode == 0
 
 
+def test_dbt_cli_subsetted_execution(mocker: MockerFixture) -> None:
+    mock_context = mocker.MagicMock()
+    mock_selected_output_names = ["least_caloric", "sort_by_calories"]
+
+    type(mock_context.op).tags = mocker.PropertyMock(return_value={})
+    type(mock_context).selected_output_names = mocker.PropertyMock(
+        return_value=mock_selected_output_names
+    )
+
+    dbt = DbtClientV2(project_dir=TEST_PROJECT_DIR)
+    dbt_cli_task = dbt.cli(["run"], manifest=manifest, context=mock_context)
+
+    dbt_cli_task.wait()
+
+    assert dbt_cli_task.process.args == [
+        "dbt",
+        "run",
+        "--select",
+        (
+            "fqn:dagster_dbt_test_project.subdir.least_caloric"
+            " fqn:dagster_dbt_test_project.sort_by_calories"
+        ),
+    ]
+    assert dbt_cli_task.process.returncode is not None
+
+
+def test_dbt_cli_default_selection(mocker: MockerFixture) -> None:
+    mock_context = mocker.MagicMock()
+    mock_selected_output_names = ["least_caloric", "sort_by_calories"]
+
+    type(mock_context.op).tags = mocker.PropertyMock(
+        return_value={
+            "dagster-dbt/select": "fqn:*",
+            "dagster-dbt/exclude": "fqn:*",
```

Comment:
this bit is confusing to me -- isn't the default exclude "None" (or more accurately, no entry in the dictionary)?

it's not really relevant to this test as we're not actually running the command, but might be confusing for someone coming across this test, maybe swap out the exclude to some fancy selection string